src/Raven.Studio/wwwroot/App/views/database/settings/databaseSettings.html

```diff
@@ -0,0 +1,231 @@
+                    <div class="col-md-7 text-center add-vertical-split"><h4><strong>Value</strong></h4></div>
+                </div>
+            </div>
+            <div class="scroll-stretch all-entries">
+                <!-- ko foreach: allEntries -->
+                <div data-bind="template: { name: 'db-settings-basic-template' }, visible: showEntry"></div>
+                <!-- /ko -->
+            </div>
+        </div>
+    </div>
+    <div class="flex-horizontal flex-stretch-items absolute-fill content-margin entries-grid" data-bind="visible: viewMode() === 'summaryMode'">
+        <div class="flex-grow flex-vertical slidein-style">
+            <div class="summary-list-container flex-vertical flex-grow" style="position:relative;">
+                <virtual-grid class="resizable virtual-grid" params="controller: summaryGridController"></virtual-grid>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script type="text/html" id="db-settings-no-matching-entries-template">
+    <div class="panel">
+        <div class="panel-body">No matching entries.</div>
+    </div>
+</script>
+
+<script type="text/html" id="db-settings-basic-template">
+    <form class="flex-form entry-basic-template" autocomplete="off">
+        <div class="panel margin-bottom margin-bottom-xs entry-content">
+            <div class="panel-body row">
+                <div class="col-md-5 add-border">
+                    <div class="form-group">
+                        <div class="margin-top margin-top-sm">
+                            <small><span data-bind="text: keyName" class="margin-right margin-right-sm"></span></small>
+                            <span class="description" data-bind="attr: { 'id': keyName }">
+                                <small class="text-info"><i class="icon-info"></i></small>
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-7 add-vertical-split">
+                    <div class="form-group no-margin">
+                        <div class="margin-top margin-top-sm">
+                            <small data-placement="left" data-toggle="tooltip" data-animation="true"
+                                   data-bind="attr: { title: hasServerValue() ? 'This is a server-wide configuration' : 'This is the default configuration' }">
+                            <!-- Waiting for icons from RavenDB-14970 -->
+                            <!-- <i class="source-icon" data-bind="css: { 'icon-xxx': hasServerValue(), 'source-icon-null': !serverOrDefaultValue() }"></i> -->
+                            <!-- <i class="source-icon" data-bind="css: { 'icon-yyy': !hasServerValue(), 'source-icon-null': !serverOrDefaultValue() }"></i> -->
+                                <span data-bind="text: serverOrDefaultValue() || 'null'" id="value"></span>
```

Comment:
you have multiple elements with name id on page!

Reply:
done - removed that id completely